flutter/flutter-provider/state-management-in-flutter-using-provider-Part2.md

```diff
@@ -0,0 +1,209 @@
+# State Management in Flutter using Provider: Part 2
+
+In previous [article](https://suresh.hashnode.dev/state-management-in-flutter-using-provider-part-1-cke8vk1cb00iny0s11lu933ch)
+we have discussed the problem of using `setState` to update the app's state in the moderate or complex app.
+
+## What we are going to know in this article
+
+1. Why we should use provider
```

Comment:
Shouldn't you talk about what is provider first?  
You can cover what is and why we use in under one heading too.  
First heading will become: "What is Provider and Why we use it" (use uppercase to show emphasis in headings)